documents-services/src/main/java/org/exoplatform/documents/rest/model/TrashElementEntity.java

```diff
@@ -31,6 +31,8 @@ public class TrashElementEntity {
 
   private String  name;
 
+  private String  path;
```

Comment:
Is this the path of the document inside the trash ? Could you update the name to make it more meaningful and differes from the originFullPath ?

Reply:
This is the node path. To avoid ambiguity between these attributes, the originFullPath has been renamed to restorePath!

Reply:
ok for me